controller/userTypeController.js

```diff
@@ -22,7 +26,28 @@ const getAll = (req, res) => {
     });
 };
 
+const retrieveById = (req, res) => {
+    const userTypeId = req.body.id;
+
+    UserType.findById(userTypeId).then(userType => {
+        if(userType){
+            return res.status(HttpStatus.OK).json(userType);
+        }else{
+            return res.status(HttpStatus.NOT_FOUND).json({
+                status: 'Error',
+                message: 'User type not found.'
+            });
+        }
+    }).catch(err => {
+        return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
+            status: 'Error',
+            message: 'There was a problem retrieving the user type.'
+        });
+    });
+}
+
 //Retrieve all user types
 module.exports = {
-    getAll: getAll
+    retrieveAll,
+    retrieveById,
```

Comment:
We should discuss internally function naming. For example: you are using `retrieve`, while me and @zirlman are using `get`. We should come to an agreeing how to name them to have a single coding style present in the project.

Reply:
I will rename my functions to match your function names, now and in the future.